buildenv/jenkins/copyrightCheck.groovy

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+Boolean FAIL = false
+String SRC_REPO = 'https://github.com/${ghprbGhRepository}.git'
+def BAD_FILES = []
+String HASHES = '###################################'
+
+timeout(time: 6, unit: 'HOURS') {
+    stage('Copyright Check') {
+        node ('worker') {
+            timestamps {
+                checkout changelog: false, poll: false,
+                        scm: [$class: 'GitSCM',
+                            branches: [[name: sha1]],
+                            doGenerateSubmoduleConfigurations: false,
+                            extensions: [[$class: 'CloneOption',
+                                            depth: 0,
+                                            noTags: false,
+                                            reference: '/home/jenkins/openjdk_cache',
```

Comment:
Just wondering what `extensions` is for?  Is openjdk_cache relevant in OMR?

Reply:
Its a repo cache that the openj9 machines have. It stores git metadata to make cloning faster. The omr machines could use it if we populated it with the omr repo on our machines. At the moment I don't believe they have it.

Reply:
Added omr to the cache on the "proxy" node where this will run.